src/pybind/mgr/dashboard/controllers/cluster_upgrade.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+
+from typing import Dict, List, Optional
+
+from ..services.exception import handle_orchestrator_error
+from ..services.orchestrator import OrchClient, OrchFeature
+from . import APIDoc, APIRouter, CreatePermission, Endpoint, EndpointDoc, \
+    ReadPermission, RESTController, UpdatePermission, allow_empty_body
+from .orchestrator import raise_if_no_orchestrator
+
+
+@APIRouter('/cluster/upgrade')
+@APIDoc("Get Cluster Details", "Cluster")
+class ClusterUpgrade(RESTController):
+    @RESTController.MethodMap()
+    @raise_if_no_orchestrator([OrchFeature.UPGRADE_LIST])
+    @handle_orchestrator_error('upgrade')
+    @EndpointDoc("Get the available cluster upgrade list",
+                 parameters={
+                     'image': (str, 'Ceph Image'),
+                     'tags': (bool, 'Show all image tags'),
+                     'show_all_versions': (bool, 'Show all available versions')
+                 })
+    @ReadPermission
+    def list(self, tags: bool, image: Optional[str] = None,
```

Comment:
Let's make the `tags` False by default. We can follow the CLI method and display the version by default.

```suggestion
    def list(self, tags: bool = False, image: Optional[str] = None,
```